node/utils/attributes.ts

```diff
@@ -186,22 +170,36 @@ const convertValues = (
   throw new Error(`Not recognized attribute type: ${attribute.type}`)
 }
 
+export const buildHref = (
+  baseHref: string,
+  key: string,
+  value: string
+): string => {
+  if (isEmpty(key) || isEmpty(value)) {
+    return baseHref
+  }
+
+  const [path = '', map = ''] = baseHref.split('?map=')
+  const pathValues = [...path.split('/'), value].filter((x) => !isEmpty(x))
+  const mapValues = [...map.split(','), key].filter((x) => !isEmpty(x))
+
+  return `${pathValues.join('/')}?map=${mapValues.join(',')}`
```

Comment:
I suppose there's no way for it to happen (?), but you never check whether pathValues and mapValues are both not empty at the same time

Reply:
I guess this test case would cover this issue, wouldn't it?
https://github.com/vtex-apps/search-resolver/pull/64/files/df5d988a5231c20a6760610d069aecb1dfe852eb#diff-4ec3b647f6eda70d0edb78e6069561caR285

As it's always adding `key` to the end of the `pathValues` array, and value to the end of `mapValues` array, the arrays themselves cannot be empty if `key` and `value` exists. The check for `key` and `value` is on line 178. Does this cover?

Reply:
Yeah, I guess this covers it